app/assets/v2/css/submit_bounty.css

```diff
@@ -80,6 +80,19 @@ textarea:read-only {
   height: 37px;
 }
 
+#gitcoin-tip-section .form__amount-wrapper {
+  width: 33%;
+}
+
+#gitcoin-tip-section .form__flex-group {
+  justify-content: space-between;
+}
+
+#gitcoin-tip-section .form__input,
+#gitcoin-tip-section .form__select2 {
+  height: 37px;
```

Comment:
Do we want to use `em` here? @thelostone-mc @SaptakS

Reply:
Any word?

Reply:
Yes , that would be super neat